packages/vaadin-combo-box/src/vaadin-combo-box-dropdown.js

```diff
@@ -225,16 +200,14 @@ class ComboBoxDropdown extends mixinBehaviors(IronResizableBehavior, PolymerElem
       return;
     }
 
-    if (opened) {
-      this.$.overlay.style.position = this._isPositionFixed(this.positionTarget) ? 'fixed' : 'absolute';
-      this._setPosition();
+    this._setOverlayWidth();
```

Comment:
I tried the following approach and it passes all the tests:

```js
  _openedChanged(opened, wasOpened) {
    if (opened) {
      this._setOverlayWidth();

      this._scroller.style.maxHeight =
        getComputedStyle(this).getPropertyValue('--vaadin-combo-box-overlay-max-height') || '65vh';

      this.dispatchEvent(new CustomEvent('vaadin-combo-box-dropdown-opened', { bubbles: true, composed: true }));
    } else if (wasOpened && !this.__emptyItems) {
      this.dispatchEvent(new CustomEvent('vaadin-combo-box-dropdown-closed', { bubbles: true, composed: true }));
    }
  }
```

IMO this is a bit cleaner that the above check. Not a blocker though.

Reply:
Thx, looks cleaner, changed